guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -43,17 +43,17 @@ ifeval::["{context}" == "{project-context}"]
 --foreman-proxy-dns-managed true \
 --foreman-proxy-dns-interface __eth0__ \
 --foreman-proxy-dns-zone __example.com__ \
---foreman-proxy-dns-forwarders __172.17.13.1__ \
---foreman-proxy-dns-reverse __13.17.172.in-addr.arpa__ \
+--foreman-proxy-dns-forwarders __192.0.2.1__ \
+--foreman-proxy-dns-reverse __2.0.192.in-addr.arpa__ \
 --foreman-proxy-dhcp true \
 --foreman-proxy-dhcp-managed true \
 --foreman-proxy-dhcp-interface __eth0__ \
---foreman-proxy-dhcp-range "__172.17.13.100__ __172.17.13.150__" \
---foreman-proxy-dhcp-gateway __172.17.13.1__ \
---foreman-proxy-dhcp-nameservers __172.17.13.2__ \
+--foreman-proxy-dhcp-range "__192.0.2.100__ __192.0.2.150__" \
+--foreman-proxy-dhcp-gateway __192.0.2.1__ \
+--foreman-proxy-dhcp-nameservers __192.0.2.2__ \
 --foreman-proxy-tftp true \
 --foreman-proxy-tftp-managed true \
---foreman-proxy-tftp-servername $(hostname)
+--foreman-proxy-tftp-servername _192.0.2.0_
 ----
 
 You can monitor the progress of the `{foreman-installer}` command displayed in your prompt. You can view the logs in `/var/log/foreman-installer/satellite.log`. You can view the settings used, including the `admin_password` parameter, in the `/etc/foreman-installer/scenarios.d/satellite-answers.yaml` file.
```

Comment:
These paths should use the scenario name. The `admin_password` parameter is called `initial_admin_password` for a few (upstream) releases but has also made its way into downstream.